rebar.config.script

```diff
@@ -27,25 +27,47 @@ end.
 VerList = lists:map(fun(X) -> {Int, _} = string:to_integer(X), Int end,
     string:tokens(VerString, ".")).
 
+
+ErlangTooOld = fun(Ver) -.
+    Msg = "This version of Erlang (~p) is too old for~n"
+        "use with Apache CouchDB.~n",
+    io:fwrite(Msg, [Ver]),
+    halt(1)
+end.
+
 NotSupported = fun(Ver) ->
-    io:fwrite("CouchDB does not support this version of Erlang (~p).~n", [Ver]),
-    io:fwrite("Check https://docs.couchdb.org/en/latest/whatsnew/index.html for the~n"),
-    io:fwrite("latest information on supported releases.~n"),
+    Msg = "This version of Erlang (~p) is not officially~n"
+        "supported by Apache CouchDB.~n"
+        "While we do not officially support this version,~n"
+        "there are also no known bugs or incompatibilities.~n",
```

Comment:
Same comment here re width, and below. Perhaps this needs some callouts (`***` or `===`) and maybe a pause of a few seconds? Because otherwise it'll get scrolled off and no one will ever read it.

Reply:
Oooh, good idea on pausing for a few seconds there.

Reply:
Added decoration and a pause when we're warning about 19.x. The others get the decoration but no pause given we either halt or ignore it on Travis.